xealth_client/gen_client.go

```diff
@@ -1,7 +1,6 @@
-// WARNING: You are using an OpenAPI 3.1.x specification, which is not yet supported by oapi-codegen (https://github.com/deepmap/oapi-codegen/issues/373) and so some functionality may not be available. Until oapi-codegen supports OpenAPI 3.1, it is recommended to downgrade your spec to 3.0.x
```

Comment:
This warning was removed by hand. Please fix the makefile so it comments the warning on regenerate.

Reply:
Done